www-client/firefox/firefox-62.0.2.ebuild

```diff
@@ -171,6 +171,12 @@ pkg_pretend() {
 	CHECKREQS_DISK_BUILD="4G"
 
 	check-reqs_pkg_setup
+
+	# make the use of neon conflict with clang, and additional with tc-is-clang to be dead sure. #666966
+	if use neon && use clang && tc-is-clang ; then
+		die "clang doesn't understand thumb instructions. Use gcc if you really need them"
```

Comment:
You are ignoring user comments from https://bugs.gentoo.org/666966

The important part of USE=neo seems to be `--with-fpu=neon` whereas `--with-thumb...` part is only failing for clang. So better change this into

```bash
if use neon ; then
  mozconfig_annotate '' --with-fpu=neon
  if ! tc-is-clang ; then
    mozconfig_annotate '' --with-thumb=yes
    mozconfig_annotate '' --with-thumb-interwork=no
  fi
fi
```

Reply:
I'll test that.

Reply:
looks legit to me, I've used CC/X=clang/++ with neon and your very elegant solution backported into the eclass: [firefox-60.2.1-with-neon-configure-log.gz](https://github.com/gentoo/gentoo/files/2424977/firefox-60.2.1-with-neon-configure-log.gz)